sdk/search/Azure.Search.Documents/samples/Sample07_VectorSearch.md

```diff
@@ -30,23 +31,42 @@ SearchIndex searchIndex = new(indexName)
         {
             IsSearchable = true,
             VectorSearchDimensions = modelDimensions,
-            VectorSearchConfiguration = vectorSearchConfigName
+            VectorSearchProfile = vectorSearchProfile
```

Comment:
This may be a case where we would want to retain both the old and new value given that the SDKs can support multiple API versions and the property sent over the wire changes names between versions.

Reply:
Due to significant breaking changes between the `2023-07-01-Preview` service version and the latest `2023-10-01-Preview` version, we are dropping support for the `2023-07-01-Preview` version in the SDK.